ui/src/routes/admin/users/create-user/index.tsx

```diff
@@ -195,14 +286,62 @@ const CreateUser = () => {
                 </FormItem>
               )}
             />
+            <FormField
+              control={form.control}
+              name='organizations'
+              render={({ field }) => (
+                <FormItem>
+                  <FormLabel>Organizations</FormLabel>
+                  <FormControl>
+                    <MultipleSelector
+                      {...field}
+                      placeholder='Grant the user READERS access to...'
```

Comment:
This basically is already part of the form description below. How about instead "Start typing to find organizations..."?